careers/styles/styles.css

```diff
@@ -112,6 +114,7 @@ body.careers {
   /* colors usages */
   --link-color: var(--static-color-caesars-red);
   --block-background-color: var(--static-color-pink-red);
+  --block-background-color-rgba: 164 27 40;
```

Comment:
"Hack" to use color values in rgba functions.

Reply:
or you can use rgb(164 27 40 / 1) notation